### PR TITLE
TRITON-2033 volapi v2 support

### PR DESCRIPTION
**Pull request by twhiteman**

### CODE REVIEW

File: lib/designation.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ var DEFAULT_FILTER_DOCKER_NFS_VOLUMES_AUTOMOUNT_MIN_PLATFORM =
     '20160613T123039Z';
 var DEFAULT_FILTER_NON_DOCKER_NFS_VOLUMES_AUTOMOUNT_MIN_PLATFORM =
     '20170925T211846Z';
+var DEFAULT_FILTER_VOLAPI_NFS_V2_MIN_PLATFORM = '20191106T000000Z';
```
Comment (twhiteman) [inline lib/designation.js]: This will need to be updated once Dan commits the NFS platform changes.

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "async": "1.5.2",
     "backoff": "2.5.0",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#a9ab5e8e3bb7ec014946d0f0b170eb6162c22da5",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#TRITON-2033",
```
Comment (twhiteman) [inline package.json]: This will need to change to a DAPI commit hash once the DAPI parts are committed.

File: lib/designation.js (hunk 1 above)
Comment (kusor) [reply lib/designation.js]: Still pending? Also, shouldn't the (C) notice be modified for the file?

File: package.json (hunk 2 above)
Comment (kusor) [reply package.json]: There it is, waiting for you to merge it! :-)

File: lib/designation.js (hunk 1 above)
Comment (twhiteman) [reply lib/designation.js]: Updated now.

**Pull request merged**